src/main/java/me/totalfreedom/totalfreedommod/blocking/BlockBlocker.java

```diff
@@ -97,22 +89,19 @@ public void onBlockPlace(BlockPlaceEvent event)
                 {
                     FLog.info(String.format("%s placed TNT @ %s", player.getName(), FUtil.formatLocation(event.getBlock().getLocation())));
 
-                    player.getInventory().clear(player.getInventory().getHeldItemSlot());
                 }
                 else
                 {
                     player.getInventory().setItem(player.getInventory().getHeldItemSlot(), new ItemStack(Material.COOKIE, 1));
-
                     player.sendMessage(ChatColor.GRAY + "TNT is currently disabled.");
                     event.setCancelled(true);
                 }
                 break;
             }
             case STRUCTURE_BLOCK:
-            case STRUCTURE_VOID:
             {
                 player.sendMessage(ChatColor.GRAY + "Structure blocks are disabled.");
-
+                player.getInventory().setItem(player.getInventory().getHeldItemSlot(), new ItemStack(Material.COOKIE, 1));
```

Comment:
Why?

Reply:
It clears the structure block usage, like TNT, but that got missed.

Reply:
can you elaborate on what you said? lol

Reply:
Well,if someone tries to place structure blocks, there is a little bug that it doesnt get cleared from the player inventory.

Reply:
Basically rather than clearing the item the replace it with a cookie just like when you place TNT as an OP currently.